spacy/lang/lb/__init__.py

```diff
@@ -0,0 +1,36 @@
+# coding: utf8
+from __future__ import unicode_literals
+
+from .tokenizer_exceptions import TOKENIZER_EXCEPTIONS 
+from .norm_exceptions import NORM_EXCEPTIONS
+from .punctuation import TOKENIZER_INFIXES
+from .lex_attrs import LEX_ATTRS
+from .tag_map import TAG_MAP
+from .stop_words import STOP_WORDS
+from .lemmatizer import LOOKUP
```

Comment:
As of v2.2, the way lemmatization data is stored has changed. You can read more about this in the [release notes](https://spacy.io/usage/v2-2).

So instead of storing the lookup table in Python with the language data, it should be added to the [`spacy-lookups-data` package](https://github.com/explosion/spacy-lookups-data/) (see the repo for examples). It should be pretty straightforward: the data needs to be JSON and the data needs to be registered in the `__init__.py` and the entry points in the `setup.cfg`. Lemmatization tests should also be moved to this repo.

Reply:
I made the changes in `spacy-lookups-data` and removed `lemmatizer.py` from the language class.